### PR TITLE
PEOPLE-107 unavailable end date

### PR DESCRIPTION
**Pull request by mczuchnowski**
**Adds:**
- puts users in Unavailable tab if end date of their unavailability is the same as current date
- spec for this bug

**Jira:**
https://netguru.atlassian.net/browse/PEOPLE-107

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -209,4 +209,21 @@ def time(year, month, day)
...
+    let!(:unavailable_user) { create(:user) }
+    let!(:unavailable_membership) do
+      create(:membership,
+        starts_at: 3.days.ago, user: unavailable_user, project: unavailable_project)
+    end
+
+    context "when current date is the same as end date for unavailable project membership" do
+      before { unavailable_membership.update_attribute(:ends_at, Date.today) }
+
+      it "returns the user who ends unavailable project today" do
```
Comment (netguru-hound) [inline spec/models/user_spec.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -209,4 +209,21 @@ def time(year, month, day)
...
+
+  describe "#unavailable" do
+    let!(:unavailable_project) { create(:project, name: "unavailable", internal: true) }
+    let!(:unavailable_user) { create(:user) }
+    let!(:unavailable_membership) do
+      create(:membership,
+        starts_at: 3.days.ago, user: unavailable_user, project: unavailable_project)
+    end
+
+    context "when current date is the same as end date for unavailable project membership" do
```
Comment (netguru-hound) [inline spec/models/user_spec.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -209,4 +209,21 @@ def time(year, month, day)
       end
     end
   end
+
+  describe "#unavailable" do
+    let!(:unavailable_project) { create(:project, name: "unavailable", internal: true) }
```
Comment (netguru-hound) [inline spec/models/user_spec.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -209,4 +209,21 @@ def time(year, month, day)
       end
     end
   end
+
+  describe "#unavailable" do
```
Comment (netguru-hound) [inline spec/models/user_spec.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/models/user_spec.rb [hunk 5]
```diff
@@ -209,4 +209,21 @@ def time(year, month, day)
...
+    let!(:unavailable_project) { create(:project, name: "unavailable", internal: true) }
+    let!(:unavailable_user) { create(:user) }
+    let!(:unavailable_membership) do
+      create(:membership,
+        starts_at: 3.days.ago, user: unavailable_user, project: unavailable_project)
```
Comment (netguru-hound) [inline spec/models/user_spec.rb]: Align the parameters of a method call if they span more than one line.

**Comment by herr-zimt** [comment]
Gut :+1:

**Pull request merged**